File: Placements.DataAccess/placement/Models/Technology.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Technology
{
    public long Id { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    public ulong IsDeleted { get; set; }

    public virtual ICollection<Companytechnology> Companytechnologies { get; set; } = new List<Companytechnology>();

    public virtual ICollection<Jobposting> Jobpostings { get; set; } = new List<Jobposting>();

    public virtual ICollection<Jobposting> JobpostingsNavigation { get; set; } = new List<Jobposting>();
}

File: web-app/src/app/services/types/Technology.ts
import { Companytechnology } from "./Companytechnology";
import { Jobposting } from "./Jobposting";

export type Technology = {
    Id: number;
    Name?: string;
    Description?: string;
    IsDeleted: boolean;
    Companytechonologies?: Companytechnology[];
    Jobpostings?: Jobposting[];
    JobpostingsNavigation?: Jobposting[];
};
